code/src/pages/cart/CartPage.scss

```diff
@@ -0,0 +1,78 @@
+@import "@design-system";
+
+.spa-cart-page {
+  padding: spacing(10);
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    gap: spacing(5);
+  }
+
+  &__items,
+  &__order-summary {
+    padding: spacing(5);
+    border: 1px solid var(--spa-order-summary-border-color, $white);
+    border-radius: spacing(1);
+    background-color: var(--spa-order-summary-background-color, $white);
+  }
+
+  &__items {
+    width: 60%;
+
+    h1 {
+      margin-bottom: spacing(5);
+      font-size: spacing(8);
+      border-bottom: 1px solid
+        var(--spa-order-cart-title-border-color, $gray-light);
+      padding-bottom: spacing(3);
+    }
+  }
+
+  &__order-summary {
+    width: 35%;
+
+    h2 {
+      margin-bottom: spacing(5);
+      font-size: spacing(8);
+      border-bottom: 1px solid
+        var(--spa-order-summary-border-color, $gray-light);
+      padding-bottom: spacing(3);
+    }
+
+    .spa-order-summary__details {
+      p {
+        font-size: spacing(4);
+        margin: spacing(3) 0;
+      }
+
+      .spa-order-summary__row {
+        display: flex;
+        justify-content: space-between;
+
+        &.spa-order-summary__total-line {
+          font-weight: var(--spa-order-summary-font-weight, $fw-extra-bold);
+          font-size: spacing(5);

```

Comment:
Use typography variant

Reply:
> Use typography variant and bold variant property

It's `AppBox` wrapper

Reply:
just add to this elements variant subtitle 2 and bold variant extra-bold. And you can easily remove this styles
![image](https://github.com/idx-academy/spa-orders/assets/118522791/7340d51e-2498-42ae-973c-23d88a73dc7f)